docs/features/feature-guides/properties/overview.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Overview
+---
+
+import FeatureAvailability from '@site/src/components/FeatureAvailability';
```

Comment:
One super big thing that could be highlighted more intensely I think is the fact that both _Assets_ AND _Columns_ can take custom properties. I bet 50-60% of our usage will come from custom column attributes

Reply:
I added a few explicit callouts to Columns throughout